tests/e2e/performance/test_pvc_creation_performance.py

```diff
@@ -24,6 +24,32 @@ class TestPVCCreationPerformance(E2ETest):
     """
     pvc_size = '1Gi'
 
+    def samples_creation(self, samples_num, teardown_factory, pvc_size):
```

Comment:
maybe change `samples_num` to `num_of_samples` to make the variable more descriptive

Reply:
@ebenahar , good idea, done.